introspection/introspection.go

```diff
@@ -63,6 +63,9 @@ func fetch(url string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to download schema: %w", err)
 	}
+	if resp.StatusCode != 200 {
+		return nil, fmt.Errorf("got unknown http status code: %s", resp.Status)
```

Comment:
unknown or invalid?

Reply:
Good point. `invalid` is indeed more correct here. I've updated the error message.